Homework-4/Operations/ListOperations.cs

```diff
@@ -0,0 +1,139 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Homework_4.Operations
+{
+    internal class ListOperations
+    {
+        /// <summary>
+        /// Prints the menu for list operations.
+        /// </summary>
+        /// <param name="collection">The name of the collection.</param>
+        private List<int> listItems = new List<int>();
+        public void PrintMenu(String collection)
+        {
+            Console.WriteLine($"\nPlease enter a number from the given options :");
+            Console.WriteLine($"1. Add an element to the {collection}.");
+            Console.WriteLine($"2. Print all the elements present in the {collection}.");
+            Console.WriteLine($"3. Delete the first element from the {collection}.");
+            Console.WriteLine($"4. Delete the middle element from the {collection}.");
+            Console.WriteLine($"5. Delete the last element from the {collection}.");
+            Console.WriteLine($"6. Calculate the average of the elements present in the {collection}.");
+            Console.WriteLine($"7. Go back to the previous menu.");
+            Console.WriteLine($"8. Exit the application.");
+        }
+        public void Operations()
+        {
+            while (true)
+            {
+                PrintMenu("List");
+                Console.Write("Enter your choice : ");
+                if (!int.TryParse(Console.ReadLine(), out int choice))
+                {
+                    Console.WriteLine("Invalid input. Please enter a valid number.");
+                    continue;
+                }
+                switch (choice)
```

Comment:
A single class should have been there for the second menu which should have been common for both list and dict.